### PR TITLE
[UTOPIA-1107] Updated to node:20.5-bullseye-slim

### PR DESCRIPTION
**Pull request by BradyMitch**
# Description

This PR includes the following proposed change(s):

https://apps.itsm.gov.bc.ca/jira/browse/UTOPIA-1107

- Updated to node 20.5
- Changed image from alpine to bullseye-slim
- Had to change apk (alpine package manager) to apt (because bulleseye is based on debian)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Updating Testing Framework(s)
- [x] Version change

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

- Tested basic use of the application.
- No errors in console and on image building.
- Jest tests pass.

## Development Dependency Working Agreement
- [x] My code DOES NOT include the importing of new dependencies into the DPIA ecosystem
- [ ] My code DOES include the importing of new dependencies into the DPIA ecosystem
**If new dependencies are being introduced to the DPIA ecosystem:**
- [ ] The functionality of the dependency drastically reduces code complexity and makes my changes more easily maintainable and readable 
- [ ] The dependency being introduced does not contain multiple layers of nested dependencies introducing maintainability complexity to the DPIA ecosystem

## Frontend Development Changes
- [ ] N/A
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to project documentation or diagrams that reflect my changes
- [ ] New and existing unit tests pass locally with my changes
- [ ] My code follows Airbnb React Style Guidelines

## API Development Changes
- [ ] N/A
- [x] I have performed a self-review of my own code
- [ ] My code follows standards and practices outlined in the BC Government API Development Guidelines
- [ ] New and existing unit tests pass locally with my changes
- [ ] My changes includes Swagger documentation updates that reflect the changes I am introducing

## Definition of Done

Definition of Done

### CODE REVIEW

**Comment by NoorChasib** [comment]
webapp docker build check is failing, is that going to be an issue?

File: src/backend/.docker/Dockerfile [hunk 1]
```diff
@@ -1,19 +1,13 @@
-FROM node:18.9-alpine
+FROM node:20.5-bullseye-slim
```
Comment (antsand) [inline src/backend/.docker/Dockerfile:1]: is there a performance difference between alpine and bullseye-slim? I guess slim is more lightweight and seems to be based on ubuntu.

File: src/backend/.docker/Dockerfile.local [hunk 2]
```diff
@@ -1,25 +1,25 @@
...
+
+# Update packages and install the required dependencies
+RUN apt-get update && \
+    apt-get install -y \
+    chromium \
+    freetype2-demos \
+    libharfbuzz0b \
+    ca-certificates \
+    fonts-freefont-ttf && \
+    apt-get clean && \
```
Comment (antsand) [inline src/backend/.docker/Dockerfile.local:11]: any benefit from cleaning and removing apt-get? since this is local would this be necessary?

**Review by antsand — APPROVED** [review]

File: src/backend/.docker/Dockerfile (hunk 1 above)
Comment (BradyMitch) [reply src/backend/.docker/Dockerfile:1]: Yes! Lightweight so less vulnerabilities, less space, better performance.

File: src/backend/.docker/Dockerfile.local (hunk 2 above)
Comment (BradyMitch) [reply src/backend/.docker/Dockerfile.local:11]: It's best practice and this is replacing the previous functionality of `--no-cache`

**Comment by BradyMitch** [comment]
> webapp docker build check is failing, is that going to be an issue?

@NoorChasib Ya it might be related to the typescript error that is fixed in my new pr

**Review by NoorChasib — APPROVED** [review]
looks great!

**Pull request merged**